bot/cogs/booster.py

```diff
@@ -21,15 +28,33 @@ async def on_connect(self) -> None:
     async def _monthly_booster(self) -> None:
         date = datetime.now(pytz.timezone('Asia/Jakarta'))
         if date.day == 1:
-            channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            admin_channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            tatsu_log_channel = self.bot.get_channel(TATSU_LOG_CHANNEL_ID)
             guild = self.bot.get_guild(GUILD_ID)
             role = guild.get_role(BOOSTER_ROLE_ID)
+            month = date.strftime("%B")
             members_content = ''
             for member in role.members:
+                await TatsuApi().add_score(member.id, 0, BOOSTER_MONTHLY_EXP)
+                embed = discord.Embed(
+                    title="<a:checklist:1077585402422112297> Score updated!",
+                    description=f"Successfully awarded `{BOOSTER_MONTHLY_EXP}` score to <@{member.id}>",
+                    timestamp=datetime.now(),
+                    colour=0x17A168,
+                )
+                embed.set_footer(
+                    text="Warnet",
+                    icon_url="https://cdn.discordapp.com/attachments/761684443915485184/1038313075260002365/warnet_logo_putih.png",
+                )
+                await tatsu_log_channel.send(embed=embed)
```

Comment:
Would be effective and efficient if the embed is sent after we added all scores. Gather all successful actions and don't forget to note any failed actions just in case to find out which users haven't gotten scores yet

<img width="491" alt="image" src="https://github.com/warnet-gi/WarnetBot/assets/20255031/3427340f-0698-4f08-a628-dd39e6f61e43">